tests/components/epic_games_store/test_config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Test the Epic Games Store config flow."""
+from http.client import HTTPException
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.epic_games_store.const import CONF_LOCALE, DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .const import DATA_ERROR_WRONG_COUNTRY, DATA_FREE_GAMES, MOCK_LOCALE
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] is None
+
+    with patch(
+        "homeassistant.components.epic_games_store.config_flow.EpicGamesStoreAPI.get_free_games",
+        return_value=DATA_FREE_GAMES,
+    ), patch(
+        "homeassistant.components.epic_games_store.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
```

Comment:
I suggest that you move `mock_setup_entry` to an auto-use fixture in `conftest.py`.
See an example in https://github.com/home-assistant/core/pull/88951/files

Reply:
This is the only place where it's mocked like that, I don't think we need that for now.

Reply:
It is the new "best practise"
It is too easy to forget it and then later it cause flaky tests, etc.